backend/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node index",
-    "dev:start": "nodemon index",
+    "dev:start": "nodemon index.js",
```

Comment:
This change wasn't really necessary, but thanks for this, you should have also made changes for the start command